api/src/server.rs

```diff
@@ -324,17 +347,17 @@ where
 
     pub async fn get_events_event_id(
         &self,
-        event_id: String,
+        event_cid: String,
     ) -> Result<EventsEventIdGetResponse, ErrorResponse> {
-        let decoded_event_id = decode_event_id(&event_id)?;
-        match self.model.value_for_key(&decoded_event_id).await {
+        let decoded_event_cid = decode_cid(&event_cid)?;
```

Comment:
we could update this to return `Ok(EventsEventIdGetResponse::BadResponse("Invalid CID"))` rather than a 500 .. but it's not doing it yet so it's fine to leave too

Reply:
oh, I have an idea for backward compatibility too.. we could try as a CID and if fails, as an EventID. This would keep the tests passing and give us a "debug" route if we have recon keys.

I was going to suggest deleting the `value_for_order_key` trait behavior, since we're not exposing an API for that.. but if we have it, supporting both would be nice (at least for a while).